src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,6 +124,37 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
+        
+        /**
+         * Represents the 'Export' button	

```

Comment:
Code comments are encouraged, I believe it's obvious what's going on for these two lines though. If you feel it's worth leaving then simply convert to a single line vs block, i.e.: `// Your comment`